querybook/webapp/redux/queryReview/action.ts

```diff
@@ -19,13 +20,22 @@ import {
 } from './types';
 
 export const fetchMyReviews =
-    (): ThunkResult<void> => async (dispatch: Dispatch<QueryReviewAction>) => {
+    (page: number = 0): ThunkResult<Promise<void>> =>
+    async (dispatch: Dispatch<QueryReviewAction>) => {
         try {
             dispatch({ type: FETCH_MY_REVIEWS_REQUEST });
-            const reviews = await QueryReviewResource.getReviewsCreatedByMe();
+            const reviews = await QueryReviewResource.getReviews(
```

Comment:
we don't need fetchMyReviews and fetchAssignedReviews, it can be the same function with a parameter of ReviewType